configs/vue3.js

```diff
@@ -1,11 +1,13 @@
 module.exports = {
   extends: [
+    require.resolve('./_vue-generic'),
```

Comment:
We did put this last an purpose, in order to overwrite rules from the preceding plugins. Does that still work?

See: https://github.com/leukeleu/eslint-config/blob/main/configs/_vue-generic.js and https://github.com/leukeleu/eslint-config/blob/main/configs/javascript.js

Reply:
The changes in this PR seem to have the desired behavior (all the rules are applied to the project as expected), I will however [read up on it](https://eslint.org/docs/latest/use/configure/configuration-files#extending-configuration-files) and make sure it's done properly and I'll let you know the results

Reply:
@jaap3 I refactored the base configs a little to use the typescript parser by default (since all javascript is valid typescript anyway). This way we don't run into issues where if the typescript parser is somehow overridden or included in the wrong way, the whole thing stops working. Now all settings are applied as expected, with the latest extended config taking priority without anything breaking.